### PR TITLE
7896 option levels bug

### PR DESCRIPTION
**Pull request by jpdimond**

### CODE REVIEW

File: app/assets/javascripts/legacy/views/option_set_form.js [hunk 1]
```diff
@@ -166,9 +166,6 @@
 
     // show/hide the option levels field
     self.option_levels_field.show(checked)
-
-    // enable/disable nested options
-    self.options_field.list.allow_nesting(checked)
```
Comment (jpdimond) [inline app/assets/javascripts/legacy/views/option_set_form.js]: This isn't something that should depend on whether levels are enabled or not.

File: app/assets/javascripts/legacy/views/option_levels_field.js [hunk 2]
```diff
@@ -24,6 +24,9 @@
         'edit': I18n.t('option_set.edit_option_level')
       }
     });
+
+    // No nesting for option levels
+    self.list.allow_nesting(false);
```
Comment (hooverlunch) [inline app/assets/javascripts/legacy/views/option_levels_field.js]: So I'm assuming this fixes it?

File: app/assets/javascripts/legacy/views/option_set_form.js (hunk 1 above)
Comment (hooverlunch) [reply app/assets/javascripts/legacy/views/option_set_form.js]: Won't this disable nested options? Note this is being called on `options_field` and not `option_levels_field`.

File: app/assets/javascripts/legacy/views/option_levels_field.js (hunk 2 above)
Comment (jpdimond) [reply app/assets/javascripts/legacy/views/option_levels_field.js]: yup!

File: app/assets/javascripts/legacy/views/option_set_form.js (hunk 1 above)
Comment (jpdimond) [reply app/assets/javascripts/legacy/views/option_set_form.js]: ah you are right, i put it back in, and the other fix still works!

**Pull request merged**